### PR TITLE
Source Gitlab: certify to beta

### LINKED ISSUE #14152: [EPIC] Source Gitlab to Beta
### According to [checklist](https://docs.google.com/spreadsheets/d/1vgh-0Crm68oEhrVdDMoR5fD5tG1uMoqkQKNeceb8r04/edit#gid=1288588270):

- [MLP](https://docs.google.com/document/d/1u95XzZ_aI2HtmGyrOdtlCu4hmEPdKAo0AEg1uOQRmro/edit#heading=h.22fd5raysmm3) improvements:

- All outstanding bug reports blocking Beta:
   - [x] https://github.com/airbytehq/airbyte/issues/12901
   - [x] https://github.com/airbytehq/airbyte/issues/20022
   - [x] https://github.com/airbytehq/airbyte/issues/12900
   - [x] https://github.com/airbytehq/airbyte/issues/11248
   - [x] https://github.com/airbytehq/airbyte/issues/21076 

<details><summary><strong>Acceptance criteria </strong></summary>

- [x] Create a checklist document that should rbe placed on the [Google Drive](https://drive.google.com/drive/folders/1qOzblo8xiBJc8AiK4AyuDTC1Ilt-GFzu)
- [x] Checklist filled up by engineering team
- [x] Collect and list all tech debt-related issues in the section below
- [x] Create/Revise MLP document and place it [here](https://github.com/airbytehq/airbyte/issues/20507)
- [x] Move checklist to the unblockers team
- [x] Reviewed/Updated by unblockers team
- [x] Move checklist to the product team for review
- [x] Reviewed/Updated by product team
- [x] Update connector documentation according to the comments in the checklist
- [x] Certification completed
 
</details>

### PR DESCRIPTION
**Pull request by davydov-d**
## What
 - Certify Gitlab Source to Beta

## How
 - Update SATs config
 - Update input config
 - Update docs

**This PR does not actually include a `release_stage` update since there is another PR which is blocked and the connector can not be in Beta until that PR is merged. So I'm gonna update `release_stage` in that PR.**

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]
Please bump the version and update the changelog.

**Review by arsenlosenko — APPROVED** [review]

**Comment by davydov-d** [comment]
> Please bump the version and update the changelog.

These changes are non-functional, they only affect acceptance tests and docs. No changes in the image - no sense to bump version

File: airbyte-integrations/connectors/source-gitlab/acceptance-test-config.yml [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
+    tests:
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.txt"
+      - config_path: "secrets/config_with_ids.json"
+        empty_streams:
+          - name: "epics"
+            bypass_reason: "Group in this config does not have epics. This stream is tested in the above TC."
+          - name: "epic_issues"
+            bypass_reason: "Group in this config does not have epics issues. This stream is tested in the above TC."
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-gitlab/acceptance-test-config.yml:26]: 👍

**Review by alafanechere — APPROVED** [review]
Good job!

**Pull request merged**